Mirror/Runtime/NetworkWriter.cs

```diff
@@ -25,7 +25,10 @@ public byte[] ToArray()
         {
             writer.Flush();
             byte[] slice = new byte[Position];
-            Array.Copy(((MemoryStream)writer.BaseStream).ToArray(), slice, Position);
+
```

Comment:
a better way is to just do:
```return writer.ToArray();```  
instead of all of this.

Reply:
Wouldn't that ignore the current position in the stream? (per the comment above the method)

Reply:
Yes it would.
As of now,  we never reset the Position,  Position is always the same as Length.

In fact I have been trying unsuccessfully to convince vis to completely remove Position,  it is no longer used anywhere, and in my opinion it breaks the abstraction.

You don't need to change the PR,  I simply posted my opinion.